lib/features/search/email/presentation/search_email_controller.dart

```diff
@@ -282,36 +251,92 @@ class SearchEmailController extends BaseController
         }
       }
     );
+
+    emailUIActionWorker = ever(
+      mailboxDashBoardController.emailUIAction,
+      (action) {
+        if (action is RefreshChangeEmailAction) {
+          _refreshEmailChanges(newState: action.newState);
+        }
+      },
+    );
   }
 
-  void _onSearchTextInputListener() {
-    if (textInputSearchFocus.hasFocus) {
-      searchIsRunning.value = false;
+  void _refreshEmailChanges({jmap.State? newState}) {
```

Comment:
In mobile, We will have `thread controller` and `search_email_controller` stay together?

Reply:
That's right.

Reply:
So we will have to queue handler for email? Try to avoid it

Reply:
No, we have separated the handling for these 2 controllers. They manage different email lists on each controller. So we need 2 web socket queue managers to handle each one.